building.html

```diff
@@ -151,11 +151,49 @@ <h3 id="building-windows">Building on Windows</h3>
       </ol>
 
 
-      <h3 id="building-nix">Building on macOS &amp; Linux</h3>
+      <h3 id="building-mac">Building on macOS</h3>
 
-      <p>Linux and macOS are <strong>not officially supported targets</strong>, and the game client does not compile. However, several of the tools can be built and run.</p>
+      <p>Compiling on macOS requires macOS version 10.14 or newer, and the Xcode build system. Xcode is available as a free download from the <a href="https://apps.apple.com/us/app/xcode/id497799835?mt=12/">App Store</a>.</p>
```

Comment:
Compiling requires Xcode 12.2. In turn - Xcode 12.2 requires macOS 10.15.4.

The resulting output will run on 10.14 - but it would be inaccurate to say you can compile from 10.14.

If we made MSL 2.3 an optional part of the build - we could get this lower. But for now Xcode 12.2 and the macOS 11 SDK are required.